pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sourcehut.
+> Note: Use `hg` in place of `git` below if using Mercurial. See `man hut` for additional details.
+> More information: <https://man.sr.ht>.
+
+- Initialize `hut`'s configuration file:
+
+`hut init`
+
+- List Git/Mercurial repositories:
+
+`hut {{git|hg}} list`
+
+- Create a Git/Mercurial repository:
+
+`hut {{git|hg}} create`
```

Comment:
`man hut`
>  create \<name\> [options...]
           Create a repository.

> -v, --visibility <string>
               Visibility to use (public, unlisted, private). Defaults to
               public.
```suggestion
- Create a public Git/Mercurial repository:

`hut {{git|hg}} create {{name}}`
```